### PR TITLE
BAR CHART: Represents time spent on all projects (Projects vs Hours)

### PR DESCRIPTION
**Pull request by iCHAIT**

### CODE REVIEW

**Comment by iCHAIT** [comment]
@Kennric This is ready for review!

File: app.rb [hunk 1]
```diff
@@ -45,7 +51,36 @@
...
+  times = ts.get_times
+  projects = ts.get_projects
+
+  times.each do |time|
+    slug = time['project'][0]
+    name = find_project(projects, slug)
+    duration = time['duration']
+
+    # Convert all durations to seconds
+    unless duration.is_a? Integer
```
Comment (Kennric) [inline app.rb]: I think rymesync should already return all durations as seconds, no?

**Comment by Kennric** [comment]
This looks good other than a possible redundant duration_to_seconds

File: app.rb (hunk 1 above)
Comment (iCHAIT) [inline app.rb]: rimesync return all durations in seconds only, but when I do `gt` on `climesync` I get all times in `"HhMm"` format as a result I need to check if its in seconds or not and if its not I convert it.

So I need not place a check when I am using timesync-node as server but if I play around with the times returned from the production server this checks is necessary. I think there is some conflict in the way time is stored in production server?

Comment (iCHAIT) [inline app.rb]: @kennric ^

**Comment by Kennric** [comment]
@iCHAIT Looks like you need another rebase here

**Comment by iCHAIT** [comment]
@Kennric  Done.

**Pull request merged**